README.md

```diff
@@ -164,6 +164,17 @@ class Store {
 }
 ```
 
+## MobX v6
+
+In order to become compatible with modern ES standards, decorators are not used by default in MobX v6. It still supports decorators, but they are not recommended for greenfield projects and require some extra setup as opposed to previous versions.
+[Read More](https://michel.codes/blogs/mobx6)
+
+- In order to use MobX 6 with decorators `makeObservable(this)` should be added to the constructor, and "useDefineForClassFields": true should be added to tsconfig.json.
+- Note that for decorator-less syntax, `makeAutoObservable` does not work with @Injectable stores
```

Comment:
Could you describe why it doesn't work with @Injectable and what to use instead?

Reply:
@kubk 

I still haven't figured out what exactly causes this issue.  makeAutoObservable cannot be used on classes that have super or are subclassed. 

For some reason, it considers classes decorated with the @Injectable to be super/subclassed. Need to dig a little deeper to understand this.

For now the more explicit makeObservable, works fine in @Injectable stores.